pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/KitRequestShipping.java

```diff
@@ -1692,7 +1693,7 @@ public static void reactivateKitRequest(Integer dsmKitRequestId, String message,
 
             long dsmKitId = KitRequestShipping.writeNewKit(dsmKitRequestId, kitRequestShipping.getEasypostAddressId(), message, false);
             kitRequestShipping.setDsmKitId(dsmKitId);
-
+            kitRequestShipping.setKitTypeName(kitRequestDao.getKitTypeByKitRequestId(dsmKitRequestId));
```

Comment:
There is not a way to get this in the query above (SQL_SELECT_KIT...)?

Reply:
SQL was used elsewhere too and we needed sampleType is re-activation code too, hence felt its cleaner this way.

Reply:
But can you add those fields to the query and just ignore the additional fields in other usages?

Reply:
refactored